apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/DataAdapterTests.cs

```diff
@@ -35,7 +35,7 @@ public DataAdapterTests(TestDatabaseFixture testFixture, ITestOutputHelper outpu
             TestLogger.TestOutputHelper = outputHelper;
         }
 
-#if NET45 || NET451 || NET452
+#if NET452 || NETCOREAPP2_0
```

Comment:
Maybe just #if !NETCOREAPP1_0 if we're expecting future TFMs to support it?

(This applies to a whole bunch of things below.)

Reply:
*seems* safe enough and more future proof.